Dockerfile

```diff
@@ -1,5 +1,7 @@
-FROM python:3
+FROM python:3.10.5
+
 COPY dist/concourse-pypi-resource-*.tar.gz .
-RUN pip install concourse-pypi-resource-*.tar.gz && \
+RUN pip install --upgrade pip && \
```

Comment:
Actually this is wrong. We should not update pip. It should match the version in lockfile, but this is not currently the case.